modules/migration-mtc-release-notes-1-8.adoc

```diff
@@ -0,0 +1,73 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8_{context}"]
+= {mtc-full} 1.8 release notes
+
+[id="new-features-and-enhancements-1-8{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+This release has no new features and enhancements.
+
+[id="deprecated-features-1-8_{context}"]
+== Deprecated features
+
+The following features are deprecated:
+
+This release has no deprecated features.
+
+[id="resolved-issues-1-8_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+.Indirect migration is stuck on backup stage
+
+In previous releases, indirect migration becomes stuck at the backup stage, due to invalid image name error.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2233097[*BZ#2233097*])
+
```

Comment:
2 suggestions:
a) General comment for this section: since we are talking about past releases, I think the sentence should be written in the past tense. 
b) Suggested rewrite: "In previous releases, an indirect migration became stuck at the backup stage, due to an invalid image name error." If the Jira includes the exact name of the error, use it  --> "..due to an `InvalidImageName` error."

Reply:
@RichardHoch Thank you. Good advice.